.github/workflows/ci-core-reusable.yml

```diff
@@ -119,20 +118,20 @@ jobs:
           mv solc-linux-amd64-v0.8.21+commit.d9974bed $(pwd)/etc/solc-bin/0.8.21/solc
           chmod +x $(pwd)/etc/solc-bin/0.8.21/solc
 
-          mkdir -p $(pwd)/etc/zksolc-bin/v1.3.14
```

Comment:
can we add variable to not change in n places?

Reply:
let's takle this in a separate PRs, this one is already too big